build.gradle

```diff
@@ -394,4 +394,24 @@ subprojects {
             }
         }
     }
+
+    // For projects that depend on gRPC during test execution, make sure to
+    // also configure ALPN if running on a platform (e.g. FreeBSD) that is not
+    // supported by io.netty:netty-tcnative-boringssl-static:jar. Also see:
+    // https://github.com/grpc/grpc-java/blob/master/SECURITY.md#tls-with-jdk-jetty-alpnnpn
+    if (project.name in ['opencensus-exporter-stats-stackdriver',
+                         'opencensus-exporter-trace-stackdriver']) {
+        def os = org.gradle.internal.os.OperatingSystem.current()
+        if (!os.isLinux() && !os.isWindows() && !os.isMacOsX()) {
+            configurations {
+                alpn
+            }
+            dependencies {
+                alpn 'org.mortbay.jetty.alpn:jetty-alpn-agent:2.0.7'
```

Comment:
Should this dependency only be added for the tests?

Reply:
Yes, it should only be added for tests. Here's how:

The dependency declaration on the ALPN agent uses a separate configuration ("alpn"), thus this dependency is not added to any classpath. The only place where the dependency is used is in the test task below:

```
test {
    jvmArgs "-javaagent:${configurations.alpn.asPath}"
}
```

Reply:
Thanks.  That makes sense.